src/sentry/templates/sentry/toolbar/login-success.html

```diff
@@ -15,28 +15,23 @@
 
     {% script %}
     <script>
-      (function() {
-        function notifyOpener() {
-          if (!window.opener) {
-            return;
-          }
+      // If the popup doesn't close, we can let people try to close it manually.
+      document.getElementById('close-popup').addEventListener('click', () => {
+        window.close();
+      });
 
-          window.opener.postMessage({
-            source: 'sentry-toolbar',
-            message: 'did-login',
-          }, '*');
-        }
+      if (!window.opener) {
+        return;
```

Comment:
does `return` work at the top level of a <script> ? TIL

Reply:
ya, it just bails

Reply:
ah, i lied it complains